sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Breaking Changes
 
 - Removed `Azure::Storage::Common::PackageVersion`.
+- Moved `HttpGetter` and `HttpGetterInfo` to internal namespace.
```

Comment:
Since this is now "hidden" from the user, we just say it is removed. Up to you.
```suggestion
- Removed `HTTPGetter` and `HttpGetterInfo` by moving them to the `_internal` namespace.
```